discovery-provider/src/queries/get_skipped_transactions.py

```diff
@@ -126,3 +131,20 @@ def set_indexing_error(
 
 def clear_indexing_error(redis_instance):
     redis_instance.delete(INDEXING_ERROR_KEY)
+
+
+def save_and_get_skip_tx_hash(session, redis):
+    """Fetch if there is a tx_hash to be skipped because of continuous errors
+    """
+    indexing_error = get_indexing_error(redis)
+    if isinstance(indexing_error, dict) and 'has_consensus' in indexing_error and indexing_error['has_consensus']:
+        num_skipped_tx = session.query(func.count(SkippedTransaction.id)).scalar()
+        if num_skipped_tx >= MAX_SKIPPED_TX:
+            return None
```

Comment:
do we still wanna save the skipped tx hash even if we're past the max amount allowed? or it's not needed for later?